schema/project-level/project-schema.json

```diff
@@ -1426,6 +1448,141 @@
+          "description": "The title of this metric",
+          "type": [
+            "string"
+          ],
+          "minLength": 1
+        },
+        "description": {
+          "title": "Description",
+          "description": "A short description of the metric. This may include short details of measurement methods.",
+          "type": [
+            "string"
+          ],
+          "minLength": 1
+        },
+        "observations": {
+          "type": "array",
+          "title": "Observations",
+          "description": "An array of target or actual values for this metric.",
+          "items": {
+            "$ref": "#/definitions/Observation"
+          },
+          "uniqueItems": true,
+          "minItems": 1
+        }
+      },
+      "minProperties": 1
+    },
+    "Observation": {
+      "type": "object",
+      "title": "Observation",
+      "description": "An actual or target observation. Observations should include either a value (for financial metrics) or measure (for other metrics).",
+      "required": [
+        "id"
+      ],
+      "properties": {
+        "id": {
+          "title": "Identifier",
+          "description": "A local identifier for this specific observation. This may be an arbitrary identifier, or could be a composite of the metric identifier, and the date and other dimensions of this observation.",
+          "type": "string",
+          "minLength": 1
+        },
+        "period": {
+          "title": "Period",
+          "description": "The period over which this observation is measured.",
+          "$ref": "#/definitions/Period"
+        },
+        "value": {
+          "title": "Value",
+          "description": "For financial metrics, the value of this forecast, target or actual observation.",
+          "$ref": "#/definitions/Value"
+        },
+        "measure": {
+          "title": "Measure",
+          "description": "For non-financial metrics, the measure of this forecast, target or actual observation. Measures may be provided as free text or numerical values.",
+          "type": [
+            "string",
+            "number"
+          ],
+          "minLength": 1
+        },
+        "unit": {
+          "type": "object",
+          "title": "Unit",
+          "description": "Unit",
+          "properties": {
+            "name": {
+              "type": [
+                "string"
+              ],
+              "title": "Unit name",
+              "description": "The name of the unit.",
+              "minLength": 1
+            },
+            "scheme": {
+              "title": "Scheme",
+              "description": "The list from which units of measure identifiers are taken. Use of the scheme 'UNCEFACT' for the UN/CEFACT Recommendation 20 list of \"Codes for Units of Measure Used in International Trade\" is recommended.",
+              "type": [
+                "string"
+              ],
+              "codelist": "unitClassificationScheme.csv",
```

Comment:
This codelist doesn't exist in OC4IDS, I don't think, which causes an error.

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/282/commits/bed6d33c04fbb933a81863d6521b564863294ec2